eventsourcing/src/main/java/org/axonframework/eventsourcing/eventstore/AbstractEventStorageEngine.java

```diff
@@ -350,12 +350,7 @@ public Builder snapshotFilter(SnapshotFilter snapshotFilter) {
          *                                    specifications
          */
         protected void validate() throws AxonConfigurationException {
-            if (snapshotSerializer == null) {
-                snapshotSerializer = XStreamSerializer::defaultSerializer;
-            }
-            if (eventSerializer == null) {
-                eventSerializer = XStreamSerializer::defaultSerializer;
-            }
+            // Kept for overriding
```

Comment:
Should still validate if the serializers are not null?

Reply:
Good point, will do!